### PR TITLE
Add Evaluate Measure Timeout

### PR DESCRIPTION
**Pull request by alexanderkiel**
The environment variable FHIR_OPERATION_EVALUATE_MEASURE_TIMEOUT can be used to set a evaluate measure timeout in milliseconds. The default value will be 1 hour in order to prevent overly long evaluations from slowing down Blaze. At the same time the default should not change the behaviour of normal evaluations.

### CODE REVIEW

File: docs/cql-queries.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ If you like to use the command line, please look into [this section](cql-queries
 
 ## API Documentation
 
-If yopu like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
+If you like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
```
Comment (lablans) [inline docs/cql-queries.md:14]:
```suggestion
If you'd like to use the CQL Evaluation API directly, please read the [CQL API Documentation](cql-queries/api.md).
```

File: docs/deployment/environment-variables.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ More information about distributed deployment are available [here](distributed.m
 | LOG_LEVEL                               | info                       | v0.6   | —      | one of trace, debug, info, warn or error                                                       |
 | JAVA_TOOL_OPTIONS                       | —                          | —      | —      | JVM options \(Docker only\)                                                                    |
 | FHIR_OPERATION_EVALUATE_MEASURE_THREADS | 4                          | v0.8   | —      | The maximum number of parallel $evaluate-measure executions.                                   |
+| FHIR_OPERATION_EVALUATE_MEASURE_TIMEOUT | 3600000 (1h)               | v0.19  | —      | Timeout in milliseconds for $evaluate-measure executions.                                      |
```
Comment (lablans) [inline docs/deployment/environment-variables.md:89]:
```suggestion
| FHIR_OPERATION_EVALUATE_MEASURE_TIMEOUT | 3600000 (1h)               | v0.19  | 3600000 | Timeout in milliseconds for $evaluate-measure executions.                                      |
```

Comment (lablans) [reply docs/deployment/environment-variables.md:89]: As noted in the PR description

**Review by lablans — APPROVED** [review]
Thank you! Just checked the documentation.

File: docs/deployment/environment-variables.md (hunk 2 above)
Comment (alexanderkiel) [reply docs/deployment/environment-variables.md:89]: That column is for the version of deprecation. The default value is already there.

File: docs/cql-queries.md (hunk 1 above)
Comment (alexanderkiel) [reply docs/cql-queries.md:14]: Thanks. I commit that change by myself, because I have changed some other minor things in between.

**Pull request merged**